cocotb/utils.py

```diff
@@ -286,7 +286,7 @@ def highlight(string, colour=ANSI.COLOR_HILITE_HEXDIFF_DEFAULT):
         if want_ansi is None:
             want_ansi = sys.stdout.isatty()  # default to ANSI for TTYs
         else:
-            want_ansi = want_ansi == '1'
+            want_ansi = want_ansi == '1' or want_ansi
```

Comment:
This matches `want_ansi == '0'`, which seems wrong

Reply:
This was supposed to match ``want_ansi == True`` which is what works for me.
I'll try to rewrite this to make the intention clearer.

Reply:
Is this better?